composer.json

```diff
@@ -13,7 +13,7 @@
         "php": ">=7.0",
         "broadway/broadway": "^2.0",
         "mongodb/mongodb": "^1.0",
-        "beberlei/assert": "^2.7"
+        "beberlei/assert": "^3.0"
```

Comment:
it was conflicting with the dependency on broadway core library: https://github.com/broadway/broadway/blob/master/composer.json#L7